source/administration/changelog.md

```diff
@@ -4,9 +4,154 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Disabled pull-to-refresh feature on Android (Chrome) to prevent unwanted page refresh.
+ - Fixed an issue with missing validation when an invalid credential was used.
+ - Fixed an issue where clicking `Save` in `Rename Channel` modal without changes did nothing.
+ - Fixed an issue where pasting files into a channel didn't work.
+ - Fixed an issue where Mattermost crashed when posting a code snippet containing white space.
+ - Fixed an issue where a validation error message did not get cleared when switching between creating a private/public channel.
+ 
+### Compatibility
+
+#### Removed and Deprecated Features
+
+- All API v3 endpoints have been deprecated, and scheduled for removal in Mattermost v5.0.
+- The permanent query parameter of the DELETE `/teams/{team_id}` APIv4 endpoint for permanently deleting a team is scheduled for removal in Mattermost v4.7. // XXX Jason needs to update
+
+#### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under `ServiceSettings` in `config.json`:
+    - Added `"ImageProxyType": ""`, `"ImageProxyOptions": ""`, and `"ImageProxyURL": ""` to ensure posts served to the client will have their markdown modified such that all images are loaded through a proxy when these keys are configured.
+    - Added `"ExperimentalGroupUnreadChannels": false` setting to show an unread channel section in the webapp sidebar.
+    - Added `"VaryByUser": false`, a user based rate limiting, to rate limit on token and on userID. // XXXX This is under `RateLimitingSettings`
+    - Added `"ExperimentalEnableDefaultChannelLeaveJoinMessages": true` that allows disabling of leave/join messages in the default channel, usually Town Square.
+
+### API Changes
+
+ - It is required that any new integrations use API v4 endpoints. For more details, and for a complete list of available endpoints, see [https://api.mattermost.com/](https://api.mattermost.com/).
+ - All API v3 endpoints have been deprecated, and scheduled for removal in Mattermost v5.0.
+
+#### RESTful API v4 Changes
+
+ - Added auto lowercase team and channel names in API requests. This ensures that the channel name is automatically lowercased for endpoints taking team or channel names as URL parameters.
+ - Added `POST /emoji/search`, `GET /emojis/name/{emoji_name}`, and `GET /emoji/autocomplete` API endpoints. These ensure that the benefits of `GET` for important performance related actions such as autocompleting are included.
+ - Added a new endpoint called `/users/tokens/search`. This endpoint gets all tokens for all users if one has the `manage_system` permission.
+
+#### Plugin API Changes (Beta)
+
+#### Plugin Hook Changes (Beta)
+
+### Database Changes
+
+### WebSocket Event Changes
+
```

Comment:
- Added `delete_team` web socket event to notify client whenever a team is deleted (e.g. via API call)

Reference:
- https://github.com/mattermost/mattermost-api-reference/pull/328/files
- https://github.com/mattermost/mattermost-server/pull/8167

Reply:
Thank you, added.